openerp/cli/scaffold.py

```diff
@@ -115,13 +115,16 @@ def render_to(self, modname, directory, params=None):
             _, ext = os.path.splitext(path)
 
             local = os.path.relpath(path, self.path)
+            # strip .template extension
+            if local.endswith('.template'):
+                local = os.path.splitext(local)[0]
```

Comment:
Maybe this could be folded in with the original splitext to avoid splitting twice?
* compute the local path first (so we always have a relative path to work with)
* splitext local into (p2, ext) (nb: should not be called p2)
* if ext is `.template`, replace local by p2